src/routes/+page.server.ts

```diff
@@ -166,6 +166,26 @@ export async function load() {
 		)
 	].sort((a, z) => a.localeCompare(z));
 
+	const mpTerms = (
+		await graphql.query({
+			organizations: {
+				__args: {
+					where: {
+						classification_EQ: 'HOUSE_OF_REPRESENTATIVE'
+					}
+				},
+				name: true,
+				term: true,
+				founding_date: true,
+				dissolution_date: true
+			}
+		})
+	).organizations
+		.filter((org) => org.name != null)
+		.sort(
+			(a, z) => new Date(z.founding_date ?? 0).getTime() - new Date(a.founding_date ?? 0).getTime()
```

Comment:
You can also use sort in graphql with `.__args.sort`. I'm not sure how GraphQL will sort `null` value, but House of Reps should always have founding date. Otherwise, we can consider as a data issue.